website/templates/project/contributors.mako

```diff
@@ -136,14 +136,13 @@
                             </div>
                         </td>
                         <td class="col-sm-4">
-                           <ul class="narrow-list list-overflow" data-bind="foreach: nodesList">
+                           <ul class="private-link-list narrow-list" data-bind="foreach: nodesList, attr: {id: 'private-link-list-' + $index() }">
```

Comment:
@caneruguz Would it be difficult to remove the constraint that osfToggleHeight must be passed elements with IDs? It seems unnecessary.